docs/multi-homing.md

```diff
@@ -234,6 +234,67 @@ spec:
 - specifying a static IP address for the pod is only possible when the
   attachment configuration does **not** feature subnets.
 
+## Multi-network Policies
+OVN-Kubernetes implements native support for
+[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy),
+an API providing
+[network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+features for secondary networks.
+
+To configure pod isolation, the user must:
+- provision a `network-attachment-definition`.
+- provision a `MultiNetworkPolicy` indicating to which secondary networks it
+  applies via the
+  [policy-for](https://github.com/k8snetworkplumbingwg/multi-networkpolicy#policy-for-annotation)
+  annotation.
+
```

Comment:
needs to set OVN_MULTI_NETWORK_ENABLE to true. Also, after https://github.com/ovn-org/ovn-kubernetes/pull/3555, needs to set new flag to true.

Reply:
Done.